base/src/main/scala/co/uproot/abandon/Report.scala

```diff
@@ -252,6 +252,9 @@ object Reports {
 
   def xmlBalanceExport(state: AppState, exportSettings: XmlExportSettings): xml.Node = {
     <abandon>
+      <version>
+        { BuildInfo.version }
+      </version>
```

Comment:
Oh, sorry, I didn't realize that the spaces in the curly braces are actual spaces in the generated xml. 
I actually though of landing `version` as an attribute to `abandon` but for some reason decided to be more straightforward.